lib/api/models/pokemon_model.freezed.dart

```diff
@@ -0,0 +1,194 @@
+// coverage:ignore-file
```

Comment:
this .freezed.dart and .g.dart files should not be committed, add these to your `.gitignore` files